docs/releases.md

```diff
@@ -30,69 +30,41 @@
-- **Annoying regressions already known:** This is a list of already known regressions to be fixed by the second part of the refactoring which will concentrate on the JSON API and front-end:
-  - _Password-dialog is shown for non-existing and private albums:_ If a user requests a URL for a non-existing or private album, the password dialog will be presented to the user as if the album was password-protected. The password dialog will be presented repeatedly to the user independent of the provided password as the authentication will never succeed.
+## Version 4
 
-_Addendum for those who migrated between 2022-01-13 and 2022-01-16:_
+### v4.5.3
 
-After the initial merge to the master branch we became aware of some performance regressions due to missing indices on the DB.
-We have patched the migration script to create the indices.
-If you migrated to the head of master after 2022-01-16 you are fine.
-If you were an early adopter and migrated between 2022-01-13 (when the original merge came out) and 2022-01-15, the indices are missing.
+Released on Aug 07, 2022
 
-Unfortunately, the migration script can only add these indices while the affected table is being created, not after table creation.
-If you are affected, you have two options:
+#### IMPORTANT
 
-1. Re-run the migration
-2. Add the missing indices manually (recommended, but does not work with SQLite and requires SQL console access)
+- The internal representation of Albums changed with version 4.5.0.  
+We strongly recommend that you **BACK UP YOUR DATABASE BEFORE UPDATING**.
+- The folder structure changed for images; **please check the required directory permissions**.  
+Read more [here  &#187;](https://lycheeorg.github.io/docs/#directory-permissions).
 
-_Option 1 (all DB engines):_ First roll back the original migration via `./artisan migrate:rollback`, update the master branch via `git pull` and re-run the migration again via `./artisan migrate`. As always make a backup of your DB first. Note, that the necessary rollback migration may not have been tested as well as the forward migration. Moreover, a repeated forward migration generates new random IDs for your photos and albums. If you are able to avoid option 1 and use option 2 instead, we recommend to use option 2.
+#### Changes
 
```

Comment:
```suggestion

- `new` : New ID scheme. Albums and photos are migrated to a new ID scheme. Previously, albums and photos used time-based integer IDs. The new IDs are truly 144bit of randomness encoded in Base64 as a 24-character string; e.g., if your photo had the URL `https://my-domain.tld/r/16102925744307/16102927818284` before, it may have the URL `https://my-domain.tld/r/GTqZfSso3nPeCnTNW4ovisgC/jM6KkmlK7X0LtVas5MjrHtTO` after the migration. This means, **external links to your albums or photos will stop working.** However, we implemented a redirection service. You can enable/disable the re-direction service under `Settings` > `More` > `legacy_id_redirection`. This gives you time to migrate any external reference to the new IDs. We also generate a log entry every time the redirection service is used. This log contains the legacy and new ID as well as information where the request came from. This may help you with the migration of your external links.

```